docs/bridge/docs/03-RFQ/index.md

```
@@ -0,0 +1,210 @@
+This includes the bridge, deposit, escrow, and eventual release of user funds on the origin chain.
+
+Associated [relay], [prove], [claim], [dispute], and [cancel] actions are also facilitated by FastBridge contracts.
+
+Additionally, the FastBridge contracts manage all relevant permissions and on-chain configuration settings of the system.
+
+FastBridge contracts will be deployed on all supported chains.
+
+
+##  Flow Summary
+
+#### [Quoting]
```

Markdownlint (0.35.0)

Expected: h3; Actual: h4
Heading levels should only increment by one level at a time

(MD001, heading-increment)